app/views/articles/_form.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= form_with model: @article do |f| %>
+
+  <div class="form-group push-down">
+    <%= f.label(:article_title, :article_title.t + ":") %>
+    <%= f.text_field(:title, class: "form-control") %>
```

Comment:
Should we parenthesize/unparenthesize some/all arguments in the views? Do we care?
- We generally parenthesize.
- No parens is less cluttered, and various Rails documentation has plenty of examples without parens. 
- RuboCop's default for `.rb` files is parens. [Style/MethodCallWithArgsParentheses](https://docs.rubocop.org/rubocop/0.85/cops_style.html#stylemethodcallwithargsparentheses). RuboCop doesn't check `.erb` files. 
- In `bootstrap-4`, @nimmolo did not use parens in views (also in many places in the tests and the app). In `bootstrap4` I tried avoiding parens when I edited the views, but often forgot about that. It's hard to switch back and forth.